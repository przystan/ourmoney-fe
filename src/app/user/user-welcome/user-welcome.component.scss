:host {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100%;
  width: 100%;
  background-image:
    linear-gradient(
      var(--primary-background-color),
      var(--secondary-background-color)
    ),
    url('/assets/images/user-background.jpg');
  background-size: cover;
}
