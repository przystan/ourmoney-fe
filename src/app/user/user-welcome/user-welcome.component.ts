import { Component, ChangeDetectionStrategy } from '@angular/core';

@Component({
  selector: 'app-user-welcome',
  template: `
    <div></div>
  `,
  styleUrls: ['./user-welcome.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class UserWelcomeComponent {}
