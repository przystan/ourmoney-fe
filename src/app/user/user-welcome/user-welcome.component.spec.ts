import { Spectator, createComponentFactory } from '@ngneat/spectator';
import { UserWelcomeComponent } from './user-welcome.component';

describe('UserWelcomeComponent', () => {
  let spectator: Spectator<UserWelcomeComponent>;

  const createComponent = createComponentFactory({
    component: UserWelcomeComponent
  });

  beforeEach(() => {
    spectator = createComponent();
  });

  it('should match snapshot', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
