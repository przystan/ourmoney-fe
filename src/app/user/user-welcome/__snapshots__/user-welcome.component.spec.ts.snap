// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserWelcomeComponent should match snapshot 1`] = `
<app-user-welcome>
  <div />
</app-user-welcome>
`;
